fdbclient/Schemas.cpp

```diff
@@ -297,7 +297,10 @@ const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
       },
       "degraded_processes":0,
       "database_available":true,
-      "database_locked":false,
+      "database_lock_state": {
+         "locked": true,
+         "lockUID": "00000000000000000000000000000000"
+      },
```

Comment:
There's also unfortunately another place that needs to be updated in the documentation which doesn't yet auto generate from this. It's in the file documentation/sphinx/source/mr-status-json-schemas.rst.inc.